### PR TITLE
INT-3953: AMQP Delay Header Mapping

### PR DESCRIPTION
**Pull request by garyrussell**
JIRA: https://jira.spring.io/browse/INT-3953

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -130,7 +130,7 @@ subprojects { subproject ->
 		tomcatVersion = "8.0.30"
 		smack3Version = '3.2.1'
 		smackVersion = '4.1.5'
-		springAmqpVersion = project.hasProperty('springAmqpVersion') ? project.springAmqpVersion : '1.5.2.RELEASE'
+		springAmqpVersion = project.hasProperty('springAmqpVersion') ? project.springAmqpVersion : '1.6.0.BUILD-SNAPSHOT'
```
Comment (artembilan) [inline build.gradle]: I guess the PR may wait for the `1.6.0.M1`.

File: spring-integration-amqp/src/main/java/org/springframework/integration/amqp/support/DefaultAmqpHeaderMapper.java [hunk 2]
```diff
@@ -70,11 +70,13 @@
 		STANDARD_HEADER_NAMES.add(AmqpHeaders.CONTENT_LENGTH);
 		STANDARD_HEADER_NAMES.add(AmqpHeaders.CONTENT_TYPE);
 		STANDARD_HEADER_NAMES.add(AmqpHeaders.CORRELATION_ID);
+		STANDARD_HEADER_NAMES.add(AmqpHeaders.DELAY);
```
Comment (artembilan) [inline spring-integration-amqp/src/main/java/org/springframework/integration/amqp/support/DefaultAmqpHeaderMapper.java]: I think we should mention these new headers for mapping in the `AMQP Message Headers` doc.
And short expression for Delayed Exchanges in the What's New would be good noise as well :smile:

**Comment by artembilan** [comment]
Only Docs concers.
We can merge it when `1.6.0.M1` will be here already.
Anyway we can't have `B-S`s for dependencies for releasing.

**Comment by garyrussell** [comment]
Pushed; agreed we can wait for M1; updated build.gradle in preparation.

**Comment by artembilan** [comment]
Since AMQP-1.6.0.M1 is out already, merging...

**Comment by artembilan** [comment]
Merged as https://github.com/spring-projects/spring-integration/commit/51a16b2f45d2c29bad6ff06f74a623f73bdae6ec

**Pull request closed without merging**